core/src/test/scala/integration/kafka/api/TransactionsExpirationTest.scala

```diff
@@ -81,9 +81,14 @@ class TransactionsExpirationTest extends KafkaServerTestHarness {
     super.tearDown()
   }
 
-  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumAndGroupProtocolNames)
-  @MethodSource(Array("getTestQuorumAndGroupProtocolParametersAll"))
-  def testBumpTransactionalEpochAfterInvalidProducerIdMapping(quorum: String, groupProtocol: String): Unit = {
+  @ParameterizedTest
```

Comment:
I think you need to use `@ParameterizedTest(name = "{displayName}.quorum={0}.groupProtocol={1}.isTV2Enabled={2}")` and leave a space after each comma in the csv list to truly disable TV2. Can you help verify?

Reply:
ummm, actually it works. interesting. Why would we even use the above formating in other tests?

Reply:
Which tests are you referring to? I think by default, the arguments are specified in order.

Reply:
Yeah Im not sure why they used the formatting in the other test